### PR TITLE
wedlocks is now able to encrypt some params

### PR DESCRIPTION
**Pull request by julien51**
# Description

This is required to ensure that the user who creates an account is actually the user
who "owns" the the email address.

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Unfortunately, it looks like CircleCi does not accept multi-line env vars easily... and we have to manually handle that.

**Comment by julien51** [comment]
Ok, found a way to add multiline stuff to CircleCi!

**Comment by julien51** [comment]
Note, before merging this we need to implement thge counter part on unlock app's side!

File: wedlocks/config.js [hunk 1]
```diff
@@ -19,6 +24,18 @@ requiredVariables.forEach(envVar => {
...
+if (process.env.BASE64_WEDLOCKS_PRIVATE_KEY) {
+  // This env variable is passed as base 64 to comply with the multiline reco by circleci:
+  // https://circleci.com/docs/2.0/env-vars/#encoding-multi-line-environment-variables
+  wedlocksPrivateKey = Buffer.from(
+    process.env.BASE64_WEDLOCKS_PRIVATE_KEY,
+    'base64'
+  ).toString('utf-8')
+}
+
+console.log(wedlocksPrivateKey)
```
Comment (cnasc) [inline wedlocks/config.js]: Just taking another look through this and noticed this `console`. Is that intentional or did it slip through?

**Pull request merged**